src/main.cpp

```diff
@@ -121,21 +121,6 @@ int main(const int argc, const char *argv[])
       return EXIT_FAILURE;
     }
 
-    if (simplify) {
-      std::cerr << "Start of initial simplification of " << inset_pos
-                << std::endl;
-      time_tracker.start("Simplification");
-
-      // Simplification reduces the number of points used to represent the
```

Comment:
Good change moving this down, this was quite easy to miss and would've been a bug in the future @nihalzp